ecs-codedeploy

```diff
@@ -91,7 +91,7 @@ function create_new_task_def_json() {
 
   # Some options in task definition should only be included in new definition if present in
   # current definition. If found in current definition, append to JQ filter.
-  CONDITIONAL_OPTIONS=(networkMode taskRoleArn placementConstraints)
+  CONDITIONAL_OPTIONS=(networkMode taskRoleArn placementConstraints executionRoleArn)
```

Comment:
So for other deployments was the execution role not being carried over in the task definition? Or what was happening previously to task definitions with execution roles?

Reply:
I do not think many repo took advantage of this deployment process `ecs-code-deploy`. As far as I know only events-api, webhooks-api, and alertsv2 are using it.
To answer your question, the repo using this deployment process does not carry executionRoleArn to the new task definition, unless they are deployed in fargate.

Reply:
Ah, okay makes sense.